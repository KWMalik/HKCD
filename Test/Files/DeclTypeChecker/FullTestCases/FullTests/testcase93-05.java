class Ni {
	public static void main(String [] args){
		
	}
}

class Knights{
	
}

class KnightsWhoSayNi extends Knights{
	
}

class KnightsWhoSayNi{
	
}