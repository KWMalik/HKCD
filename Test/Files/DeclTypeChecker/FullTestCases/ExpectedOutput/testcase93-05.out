Class named KnightsWhoSayNi already exists.
